test/integration/policy-imagemanifest_test.go

```diff
@@ -0,0 +1,158 @@
+const operatorNS = "openshift-operators"
+
+var ImageManifestLabel = Label("policy-collection", "stable")
+
+var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the "+policyIMVName+" policy", ImageManifestLabel, func() {
+	It("stable/"+policyIMVName+" should be created on the Hub", func() {
+		By("Creating the policy on the Hub")
+		_, err := utils.KubectlWithOutput(
+			"apply", "-f", policyIMVURL, "-n", userNamespace, "--kubeconfig="+kubeconfigHub,
+		)
+		Expect(err).To(BeNil())
+
+		By("Patching placement rule")
+		err = common.PatchPlacementRule(
+			userNamespace, "placement-"+policyIMVName, clusterNamespace, kubeconfigHub,
+		)
+		Expect(err).To(BeNil())
+
+		By("Checking that " + policyIMVName + " exists on the Hub cluster")
+		rootPolicy := utils.GetWithTimeout(
+			clientHubDynamic, common.GvrPolicy, policyIMVName, userNamespace, true, defaultTimeoutSeconds,
+		)
+		Expect(rootPolicy).NotTo(BeNil())
+	})
+
+	It("stable/"+policyIMVName+" should be created on managed cluster", func() {
+		By("Checking the policy on managed cluster in ns " + clusterNamespace)
+		managedPolicy := utils.GetWithTimeout(
+			clientManagedDynamic,
+			common.GvrPolicy,
+			userNamespace+"."+policyIMVName,
+			clusterNamespace,
+			true,
+			defaultTimeoutSeconds,
+		)
+		Expect(managedPolicy).NotTo(BeNil())
+	})
+
+	It("stable/"+policyIMVName+" should be NonCompliant", func() {
+		By("Checking if the status of the root policy is NonCompliant")
+		Eventually(
+			common.GetComplianceState(clientHubDynamic, userNamespace, policyIMVName, clusterNamespace),
+			defaultTimeoutSeconds*2,
+			1,
+		).Should(Equal(policiesv1.NonCompliant))
+	})
+
+	It("Enforcing stable/"+policyIMVName, func() {
+		By("Patching remediationAction = enforce on the root policy")
+		_, err := clientHubDynamic.Resource(common.GvrPolicy).Namespace(userNamespace).Patch(
+			context.TODO(),
+			policyIMVName,
+			k8stypes.JSONPatchType,
+			[]byte(`[{"op": "replace", "path": "/spec/remediationAction", "value": "enforce"}]`),
+			metav1.PatchOptions{},
+		)
+		Expect(err).To(BeNil())
+	})
+
+	It("The subscription should exist", func() {
+		By("Checking the subscription in the " + operatorNS + " namespace")
+		Eventually(
+			func() string {
+				output, _ := common.OcHub("get", "subscriptions.operators.coreos.com", "-n",
+					operatorNS, subName, "-o", "name")
+
+				return output
+			},
+			defaultTimeoutSeconds*2,
+			1,
+		).Should(Equal("subscription.operators.coreos.com/container-security-operator"))
+	})
+
+	It("The operator should exist", func() {
+		By("Checking the operator in the " + operatorNS + " namespace")
+		Eventually(
+			func() string {
+				output, _ := common.OcHub("get", "operators", subName+"."+operatorNS, "-o", "name")
+
+				return output
+			},
+			defaultTimeoutSeconds*2,
+			1,
+		).Should(Equal("operator.operators.coreos.com/container-security-operator.openshift-operators"))
+	})
+
+	It("stable/"+policyIMVName+" should detect imageManifestVulns", func() {
```

Comment:
Can you also add check for other two templates which should be `Compliant`? Because the current check are all noncompliant. This is not sufficient to ensure policy is working as expected.